manifests/a/Amazon/WorkspacesClient/5.17.0/Amazon.WorkspacesClient.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Amazon.WorkspacesClient
+PackageVersion: 5.17.0
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: '{68D21FCE-B884-45AB-8F18-15E42A8F6F32}'
+ReleaseDate: 2023-11-16
+AppsAndFeaturesEntries:
+- DisplayVersion: 5.17.0.4420
+  ProductCode: '{68D21FCE-B884-45AB-8F18-15E42A8F6F32}'
+  UpgradeCode: '{CB7B8EA8-3D5A-4233-A9CB-31A692E24E62}'
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```
Based on the error, I'm guessing this is a 64-bit application.